splitgraph/ingestion/sqlite/__init__.py

```diff
@@ -70,7 +73,16 @@ def db_from_minio(url: str) -> Generator[sqlite3.Connection, None, None]:
 
 
 # partly based on https://stackoverflow.com/questions/1942586/comparison-of-database-column-types-in-mysql-postgresql-and-sqlite-cross-map
-def sqlite_to_postgres_type(sqlite_type: str) -> str:
+def sqlite_to_postgres_type(raw_sqlite_type: str) -> str:
+    sqlite_type = raw_sqlite_type.upper()
+    match = re.search(RE_SINGLE_PARAM_TYPE, sqlite_type)
+    # parameteric types, eg: VARCHAR(255), DECIMAL(2,10) overlap with PostgreSQL types, so
```

Comment:
Is this true? e.g. `NATIVE CHARACTER(70)` doesn't have a postgresql equivalent

Reply:
Good point, that can happen:
```
sqlite> select type from pragma_table_info('datatype_tests');
INT
TEXT
varchar(255)
REAL
decimal(2,10)
decimal(20,100)
BLOB
decimal(2,    20)
```

Reply:
@mildbyte nice example re: `NATIVE CHARACTER(70)`, added logic + unit tests for handling such types.